index.html

```diff
@@ -272,29 +273,42 @@ <h5 class="navbar__submenu-title--right">Students</h5>
         <section>
             <div class="welcome">
                 <div class="welcome__box">
-                    <h1 class="welcome__title">GEEKS UI, Premium LMS / Education HTML Template</h1>
+                    <h1 class="welcome__title"><span class="welcome__geeks-span">Geeks UI</span>, Premium LMS / Education HTML Template</h1>
                     <p class="welcome__description">Geeks UI Premium LMS education HTML template, perfect for online courses, colleges, schools, training hubs, and corporate programs. Enhance your educational platform with seamless design and advanced features for modern learning.</p>
                 </div>
                 <div class="welcome__btn-box">
-                    <button class="welcome__button welcome__button--primary">Buy Geeks</button>
-                    <button class="welcome__button welcome__button--secondary">Preview</button>
+                    <button class="welcome__button welcome__button--primary welcome__button--with-icon"><i class="fa-solid fa-cart-shopping"></i><span>Buy Geeks</span></button>
+                    <button class="welcome__button welcome__button--secondary">Preview <span><i class="welcome__preview-icon fa-solid fa-arrow-up-right-from-square"></i></span></button>
```

Comment:
`--with-icon` modifier's purpose is to modify an element in context of space between icon and a text, since there are few buttons with the same styles, but some are with, and anothers without icon inside. Could you clarify if I should just rename it for better clarity or it needs to be different approach?

Reply:
'modifier' is more for a purpose of dynamically changing state of an element on the page when client interacts with it. You are just setting buttons to permanently have styles that suit them therefore using modifiers isn't the best approach. I'd do something like class="button button_primary welcome__button" or something similar, where button gives you standard styles used in all buttons on the page, button_primary sets more specific styles like color, border etc and then welcome button sets styles specific to that area.

Reply:
Alright, I see it now, thank you. I will try to apply that